wiki/pages/Testing-Packages.md

```diff
@@ -0,0 +1,17 @@
+As mentioned in other pages such as [tests](Package-Definition-Guide#tests),
+you can specify Rez tests to run like this:
+
+	]$ rez-test maya_utils lint unittest_name
+
+This example runs two tests, "lint" and "unittest_name", from the "maya_utils"
```

Comment:
```suggestion
This example runs two test targets, "lint" and "unittest_name", from the "maya_utils"
```

I think using test target instead of test is important because targets are not necessarily tests. The `rez-test` command is a little bit misnamed but that's another discussion (a discussion that we already had in the last years and that I feel we will probably have in the future too).

Reply:
> The rez-test command is a little bit misnamed but that's another discussion (a discussion that we already had in the last years and that I feel we will probably have in the future too).

Yeah, I always just pretend it's called `rez-task`, haha. rez-*test* implies that it's only for unittesting.